.npmignore

```diff
@@ -0,0 +1,8 @@
+react/
+svg/
+svg-min/
+.idea/
+codepoints.json
+exporter-multi.jsx
+social-logos.ai
+Gruntfile.js
```

Comment:
Since it seems we want very few files to be included in the npm package, the [package.json `files` directive](https://docs.npmjs.com/files/package.json#files) can be a nice alternative to specify only the files and directories you want to be _included_.